lib/pages/chat_list/chat_list.dart

```diff
@@ -139,7 +137,7 @@ class ChatListController extends State<ChatList>
   // Needs to match GroupsSpacesEntry for 'separate group' checking.
   List<Room> get spaces => activeClient.rooms.where((r) => r.isSpace).toList();
 
-  ValueNotifier<String?> activeRoomIdNotifier = ValueNotifier(null);
+  ActiveRoomIdNotifier get activeRoomIdNotifier => ActiveRoomIdNotifier();
 
   bool get isSelectMode => selectModeNotifier.value == SelectMode.select;
```

Comment:
why dont you use the varialble, if you use this, everytime it call, it will create a new one